tests/utils.py

```diff
@@ -157,11 +178,18 @@ def create_genesis(accounts):
     genesis = {}
     genesis["nonce"] = "0xdeadbeefdeadbeef"
     genesis["timestamp"] = "0x0"
-    genesis["parentHash"] = "0x0000000000000000000000000000000000000000000000000000000000000000"
+    # Start after homesteam
+    genesis["parentHash"] = (
+        "0x584bdb5d4e74fe97f5a5222b533fe1322fd0b6ad3eb03f02c3221984e2c0b430"
```

Comment:
You are right, I miscalculated as far as the way of calculating the block hash is concerned. This won't work.

So at the moment there is no way to start from a specific block. Already pinged the GO people to add this feature, or at least specify the block where the homestead change happens.